### PR TITLE
Modal List Rating

### PR DESCRIPTION
**Pull request by ApurvaSawant11**
Hello,

I have added the Modal, List, and Rating components for the Documentation site of my Component Library.

Please review and suggest.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -156,66 +162,85 @@ header {
...
+
 pre code.hljs::-webkit-scrollbar {
-    display: none; /* Chrome, Safari, Opera */
+    display: none;
+    /* Chrome, Safari, Opera */
 }
 
-.hljs{
+.hljs {
     background: #f8f8f8 !important;
```
Comment (Dev-AkshatGupta) [inline styles.css:226]: I was told that we should avoid using Important and look where is it taking its color from and then change it

**Review by anik31 — APPROVED** [review]
Nice work. The ratings are cool.

File: styles.css (hunk 1 above)
Comment (ApurvaSawant11) [reply styles.css:226]: This is for `highlightjs`. I have customized the colors for my code snippets. So it's important to add it. Else it won't overwrite `hljs` predefined styles.

Comment (Dev-AkshatGupta) [reply styles.css:226]: Ok rest is great even UI and all part was amazing

**Review by Dev-AkshatGupta — APPROVED** [review]

**Pull request merged**